runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1126,15 +1126,21 @@ TR_SharedCacheRelocationRuntime::validateAOTHeader(TR_FrontEnd *fe, J9VMThread *
    }
 
 const TR_AOTHeader *
-TR_SharedCacheRelocationRuntime::getStoredAOTHeader(J9VMThread *curThread)
+TR_SharedCacheRelocationRuntime::getStoredAOTHeaderWithConfig(J9SharedClassConfig *sharedClassConfig, J9VMThread *curThread)
    {
    J9SharedDataDescriptor firstDescriptor;
    firstDescriptor.address = NULL;
-   javaVM()->sharedClassConfig->findSharedData(curThread, aotHeaderKey, aotHeaderKeyLength,
+   sharedClassConfig->findSharedData(curThread, aotHeaderKey, aotHeaderKeyLength,
```

Comment:
Could we please align the second line of parameters like it was done in the old implementation?